tests/drum/run-drum-tests-in-container.sh

```diff
@@ -38,7 +38,7 @@ source $GIT_ROOT/tests/drum/integration-helpers.sh
 
 cd $GIT_ROOT || exit 1
 
-CMRUNNER_REQUIREMENT_PATH=$GIT_ROOT/custom_model_runner/requirements.txt
+CMRUNNER_REQUIREMENT_PATH=$GIT_ROOT/custom_model_runner/requirements_dev.txt
```

Comment:
Why?

Reply:
I added a dependency into requirements_test that I needed. I could change these to requirements_test as well...